spotinst_sdk2/test/test_lib/input/elastigroup/aws_group.json

```diff
@@ -187,8 +187,7 @@
               "targetSetId": "ts-123456789",
               "balancerId": "lb-123456789",
               "autoWeight": true,
-              "azAwareness": false,
-              "type": "MULTAI_TARGET_SET"
+              "azAwareness": false
```

Comment:
this part is not required as  it is related to multai_target_sets

Reply:
Fixed.